src/Server/RestRouterHandlers/RestRouterHandler.cpp

```diff
@@ -96,11 +96,20 @@ void RestRouterHandler::setupDistributedQueryParameters(
     {
         query_context->setQueryParameter(kv.first, kv.second);
     }
+
+    /// setup query parameter in context : a=b&c=d&e=f
+    std::vector<String> url_params;
```

Comment:
nit: can we have a different way to get the raw URL parameters like from the HTTPRequest ? we can save a copy of the raw URL parameters in base class if necessary

Reply:
Because simply k=v concatenation sometimes doesn't work.

Reply:
Here URL parameters are parsed from HTTPRequest  even it doesn't work.

Reply:
url parameters shall be escaped sometimes, like there is `=` sign at the beginning of value. Simple concatenation like something like k + '=' + v would result something like `k==abc` if value itself is `=abc`.

Reply:
HTTPRequest::getURI()::getRawQuery() may be something you like to explore